.github/workflows/scripts/unix_docker_build.sh

```diff
@@ -5,7 +5,8 @@ set -ex
 # Parse ARGs
 PY=$1
 GPU_BUILD=$2
-CI_SETUP_CMAKE_ARGS=$3
+PROJECT_NAME=$3
```

Comment:
can we make project_name optional :P (which saves us from specifying it in each pre/post submit jobs)?

Reply:
If we use one script for all, --project-name has to follow a name and cannot be empty... So I don't think this can be optional.

Reply:
I'll merge this first so tomorrow's auto release can test this. If we have better idea I'll open another PR :).